analyzers/src/SonarAnalyzer.CSharp/Rules/ParametersCorrectOrder.cs

```diff
@@ -35,6 +35,16 @@ public sealed class ParametersCorrectOrder : ParametersCorrectOrderBase<Argument
     {
         private static readonly DiagnosticDescriptor rule =
             DiagnosticDescriptorBuilder.GetDescriptor(DiagnosticId, MessageFormat, RspecStrings.ResourceManager);
+
+        private static SyntaxToken? GetExpressionSyntaxIdentifier(ExpressionSyntax expressionSyntax) =>
```

Comment:
Just to make it shorter and readable
```suggestion
        private static SyntaxToken? GetExpressionSyntaxIdentifier(ExpressionSyntax expression) =>
```

Reply:
Have made it shorter